conans/client/generators/premake.py

```diff
@@ -34,8 +34,8 @@ def content(self):
                     'conan_libdirs{dep} = {{{deps.lib_paths}}}\n'
                     'conan_bindirs{dep} = {{{deps.bin_paths}}}\n'
                     'conan_libs{dep} = {{{deps.libs}}}\n'
-                    'conan_cppdefines{dep} = {{{deps.defines}}}\n'
-                    'conan_cppflags{dep} = {{{deps.cppflags}}}\n'
+                    'conan_defines{dep} = {{{deps.defines}}}\n'
+                    'conan_cxxflags{dep} = {{{deps.cppflags}}}\n'
```

Comment:
Please use the same naming here to avoid confusion with the variable mapping and let the user make the conversion.
```suggestion
                    'conan_cppflags{dep} = {{{deps.cppflags}}}\n'
```

Reply:
I am not sure about this. In ``CMake`` helper:

```cmake
set(CONAN_CXX_FLAGS_{dep}{build_type} "{deps.cppflags}")
```

Reply:
I think it's to match with built-in `CMAKE_CXX_FLAGS`, right?
anyway, let's have a decision on `cppflags` vs `cxxflags`

Reply:
Finally we agreed on having ``cpp_info.cppflags`` converted to ``conan_cxxflags``. So no further action is required.